test/test_unit_circle.jl

```diff
@@ -0,0 +1,24 @@
+using StaticArrays
+
+@testset "stastical unit circle test" begin
+    pts = Array{Array{Float64,1},1}(5)
```

Comment:
Later you populate this array with `SVector`. So either change its eltype here or populate with plain vectors.

Reply:
@jw3126 can you help to change the correct eltype. What should I type? My skills are more on the management than coding.

Reply:
You could do `pts = Vector{SVector{2,Float64}}(5)`